### PR TITLE
Part1: Removing content related to the wallet. Start use the pip installable package.

### PR DESCRIPTION
**Pull request by basfroman**

### CODE REVIEW

File: bittensor/__init__.py [hunk 1]
```diff
@@ -1,21 +1,20 @@
 # The MIT License (MIT)
-# Copyright © 2021 Yuma Rao
-# Copyright © 2022-2023 Opentensor Foundation
-# Copyright © 2023 Opentensor Technologies Inc
-
+# Copyright © 2024 OpenTensor Foundation
```
Comment (thewhaleking) [inline bittensor/__init__.py]: Opentensor not OpenTensor

File: bittensor/mock/subtensor_mock.py [hunk 2]
```diff
@@ -1,30 +1,31 @@
 # The MIT License (MIT)
-# Copyright © 2022-2023 Opentensor Foundation
-
+# Copyright © 2024 OpenTensor Foundation
```
Comment (thewhaleking) [inline bittensor/mock/subtensor_mock.py]: Opentensor

File: bittensor/__init__.py (hunk 1 above)
Comment (basfroman) [reply bittensor/__init__.py]: This license has been through so many reviews. No one noticed before.

**Review by thealligatorking — APPROVED** [review]

**Review by thewhaleking — APPROVED** [review]
I don't want to say "LGTM" because it looks atrocious with the `# type: ignore` comments everywhere, but I understand the need to have them to preserve backwards compatibility. One day, when we're no longer hampered by this, I know you will create a beautiful new design.

**Pull request merged**